docs_source_files/content/grid/grid_4/components_of_a_grid.de.md

```diff
@@ -1,12 +1,39 @@
 ---
 title: "Komponenten"
-weight: 1
+weight: 2
 ---
 
 {{% notice info %}}
-<i class="fas fa-language"></i> Diese Seite wird von Englisch
-auf Deutsch übersetzt. Sprichst Du Deutsch? Hilf uns die Seite
+<i class="fas fa-language"></i> Diese Seite wird von Englisch 
+auf Deutsch übersetzt. Sprichst Du Deutsch? Hilf uns die Seite 
 zu übersetzen indem Du uns einen Pull Reqeust schickst!
-{{% /notice %}}
+ {{% /notice %}}
+![Grid](/images/grid.png)
```

Comment:
This is copying the same text from the Grid 3 section. This section should be written again for Grid 4.

Reply:
Hey @diemol yes, I am changing it, I just copied those files. Changing it 👍